{
  try {
    getResultSet().afterLast();
  }
 catch (  SQLException sqle) {
    throw convert(sqle,"exception calling afterLast()");
  }
}
